Helper/QueryFilterHelper.php

```diff
@@ -154,14 +155,15 @@ private function addOperatorExpression(
     /**
      * Form the logical expression needed to limit the CustomValue's value for given operator.
      *
-     * @return CompositeExpression|string
+     * @param float|bool|array<mixed>|string|null $filterParameterValue
      */
     private function getCustomValueValueExpression(
         SegmentQueryBuilder $customQuery,
         string $tableAlias,
         string $operator,
-        string $valueParameter
-    ) {
+        string $valueParameter,
+        float|bool|array|string $filterParameterValue = null,
```

Comment:
I'm afraid this plugin still needs to run on PHP 7.4 as defined in composer.json as someone from the community can run it on PHP 7.4 with Mautic 4. Can we remove the type here and add it as `mixed` in the comments?

Same with the return type.